server/utils/utils.ts

```diff
@@ -21,3 +21,9 @@ export const initialiseName = (fullName?: string): string | null => {
   const array = fullName.split(' ')
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
+
```

Comment:
If it wasn't for the fact that we already have this `utils` file in a `utils` folder 🤮  (and that it's inherited from the template) I'd propose something else 🤣 
(Theres lots of discussion on the internet that ["utils" is an antipattern](https://www.google.com/search?q=utils+is+an+antipattern) )